### ISSUE TITLE
missing files cause 'java.io.IOException: This archive contains unclosed entries.'

### ISSUE THREAD

**Issue by OlegYch** [sbt/sbt-native-packager#1035]
## Expected behaviour

`dist` errors out with 'missing file'

## Actual behaviour

`dist` fails with `java.io.IOException: This archive contains unclosed entries.`, and creates a corrupt zip file

## Information

plugins.sbt

`addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.2.2")`

build.sbt

```
enablePlugins(JavaAppPackaging)
mappings in Universal in packageBin += file("nothing") -> "nothing"
```

**Comment by muuki88** [comment]
This is also something we should integrate in #1026

I guess at the moment there is no error handling at all.

**Comment by retronym** [comment]
Fixed by #1124. Since version 1.3.9 the `dist` task for the project above fails fast with a better error message:

```
1] Not found: /Users/jz/code/native-package-test/nothing (mapped to nothing)
```

**Comment by retronym** [comment]
Also improving the underlying exception handling: https://github.com/sbt/sbt-native-packager/pull/1290/files
